node.js

```diff
@@ -49,7 +49,42 @@ module.exports = {
     'space-before-function-paren': off,
     'keyword-spacing': error,
     'eol-last': [ error, always ],
-    'brace-style': [ error, '1tbs', { allowSingleLine: true } ],
-    'curly': [ error, all ]
+    'brace-style': [ error, '1tbs', { allowSingleLine: false } ],
+    'curly': [ error, all ],
+    'block-spacing': [ error,  always ],
+    'indent': [ error, 2, { SwitchCase: 1 } ],
+    'no-tabs': error,
+    'spaced-comment': [ error, always ],
+    'array-callback-return': warn,
+    'eqeqeq': [ error, always, { null: 'ignore' } ],
```

Comment:
Enforce strict equality `===` instead of allowing type coercion `==`. It is considered good practice to use the type-safe equality operators === and !== instead of their regular counterparts == and !=. Allows `== null` for now as it's in use across a few projects (equals `null` or `undefined`). Separate discussion.
https://eslint.org/docs/rules/eqeqeq
```js
// Old
const x = 42
if (x == '42') {
  // ...
}
// New
if (x === 42) {
  // ...
}
```
*Not all of these are auto-fixable*